package-lock.json

```diff
@@ -22665,7 +22665,7 @@
         },
         "got": {
           "version": "9.6.0",
-          "resolved": "https://registry.npmjs.org/got/-/got-9.6.0.tgz",
+          "resolved": "http://registry.npmjs.org/got/-/got-9.6.0.tgz",
```

Comment:
http?

Reply:
`sed -i -e 's/http:\/\//https:\/\//g' package-lock.json`

Reply:
yeah I've ran into this before - I just ran `npm i`, had similar discussion with Stephen D here: https://github.com/alphagov/pay-frontend/pull/1828#discussion_r501548889

Reply:
It’s this joy: https://github.com/npm/npm/issues/20106

Reply:
Do I edit it or remain with `http`?

Reply:
it's probably because of local version of the package using http. if you remove node_modules and npm install again it shouldn't replace https. Edit it to https

Reply:
Nuked local and got the same result 🤔 should I just manually change it?

Reply:
run `sed -i -e 's/http:\/\//https:\/\//g' package-lock.json`!

Reply:
There is a bit more https://github.com/npm/npm/issues/20106#issuecomment-383770315 but yeah either remove it manually or `sed` ^

Reply:
went with `sed`... thanks all